### ISSUE TITLE
Non-repeating groups cause warning

### ISSUE THREAD

**Issue by smoliver** [humanmade/Custom-Meta-Boxes#349]
Non-repeating groups do not redisplay the input information after publishing.  Cause the following warning:

`Warning: Invalid argument supplied for foreach() in mu-plugins/Custom-Meta-Boxes/classes.fields.php on line 1550`

**Comment by swissspidy** [comment]
We're currently facing the same issue, but with repeatable groups.

Changing `$values = $this->get_value();` to `$values = $this->get_values();` inside `\CMB_Group_Field::html()` (see https://github.com/humanmade/Custom-Meta-Boxes/blob/0ff7b171cb5c4985dae31cfe69bb06b0923dd1b7/classes.fields.php#L1541) seems to resolve it.

**Comment by swissspidy** [comment]
Actually, this seems to be caused by the recent fix in #344.

**Comment by mikeselander** [comment]
Closing as this has been addressed in #351. Will be included in the next release.

**Comment by veganista** [comment]
I seem to be getting a similar warning now when trying to add non-repeating groups.

`Warning: Invalid argument supplied for foreach() in /test/wp-content/plugins/test/vendor/humanmade/custom-meta-boxes/classes.fields.php on line 1974`

```php
add_filter( 'cmb_meta_boxes', 'cmb_sample_metaboxes' );

function cmb_sample_metaboxes( array $meta_boxes ) {

    $group = [
        [
            'id'   => 'foo',
            'name' => 'Foo',
            'type' => 'text',
        ],
    ];

    $fields = [
        [
            'id'   => 'group_name',
            'name' => 'Bar',
            'type' => 'group',
            'fields' => $group,
            //'repeatable' => true,
        ],
    ];

    $meta_boxes[] = array(
        'title'    => 'Box',
        'pages'    => 'page',
        'context'  => 'normal',
        'priority' => 'high',
        'fields'   => $fields
    );

    return $meta_boxes;
}
```
